libraries/exoplayer/src/main/java/androidx/media3/exoplayer/upstream/Loader.java

```diff
@@ -326,10 +326,10 @@ private final class LoadTask<T extends Loadable> extends Handler implements Runn
 
     private static final String TAG = "LoadTask";
 
-    private static final int MSG_START = 0;
-    private static final int MSG_FINISH = 1;
-    private static final int MSG_IO_EXCEPTION = 2;
-    private static final int MSG_FATAL_ERROR = 3;
+    private static final int MSG_START = 1;
```

Comment:
This one is likely to cause some hidden issues as the same looper/handler is used to post runnable in the code.